docs/release_notes.rst

```diff
@@ -18,6 +18,7 @@ New Features
 * Extend support for detecting missing argument description in Numpy style
   docstrings (#407).
 * Added support for Python 3.8 (#423).
+* Allow skipping on errors on module level docstring (#427).
```

Comment:
Mention the feature name (noqa), e.g.:

Allow skipping errors on module level docstring via `#noqa` (#427).

Reply:
Done.